### PR TITLE
Mappings for new BlockEntity update packet logic

### PR DESCRIPTION
**Pull request by shartte**

### CODE REVIEW

File: mappings/net/minecraft/block/entity/BlockEntity.mapping [hunk 1]
```diff
@@ -42,6 +44,14 @@ CLASS net/minecraft/class_2586 net/minecraft/block/entity/BlockEntity
 		ARG 2 state
 	METHOD method_31664 setCachedState (Lnet/minecraft/class_2680;)V
 		ARG 1 state
+	METHOD method_38235 toUpdatePacket ()Lnet/minecraft/class_2596;
+		COMMENT Implement and return a packet that should be sent to players nearby when the observable state of
+		COMMENT this block entity changes.
+		COMMENT <p/>
```
Comment (liach) [inline mappings/net/minecraft/block/entity/BlockEntity.mapping]:
```suggestion
		COMMENT <p>
```
`<p/>` is like `<p></p>`, I suppose? We want the text in the paragraph than separate them with paragraphs.

File: mappings/net/minecraft/block/entity/BlockEntity.mapping [hunk 2]
```diff
@@ -42,6 +44,14 @@ CLASS net/minecraft/class_2586 net/minecraft/block/entity/BlockEntity
 		ARG 2 state
 	METHOD method_31664 setCachedState (Lnet/minecraft/class_2680;)V
 		ARG 1 state
+	METHOD method_38235 toUpdatePacket ()Lnet/minecraft/class_2596;
+		COMMENT Implement and return a packet that should be sent to players nearby when the observable state of
```
Comment (liach) [inline mappings/net/minecraft/block/entity/BlockEntity.mapping:48]: A question: in code, this is `@Nullable` and defaults to return `null`. Probably worth to note the `null` behavior, e.g. if it is not sent at all or sent with some other default mechanism.

File: mappings/net/minecraft/block/entity/BlockEntity.mapping [hunk 3]
```diff
@@ -42,6 +44,14 @@ CLASS net/minecraft/class_2586 net/minecraft/block/entity/BlockEntity
...
+		COMMENT this block entity changes.
+		COMMENT <p/>
+		COMMENT If the data returned by {@link #toInitialChunkDataNbt initial chunk data} is suitable for updates,
+		COMMENT the following shortcut can be used to create an update packet: {@code BlockEntityUpdateS2CPacket.create(this)}.
+		COMMENT <p/>
```
Comment (liach) [inline mappings/net/minecraft/block/entity/BlockEntity.mapping]:
```suggestion
		COMMENT <p>
```

File: mappings/net/minecraft/block/entity/BlockEntity.mapping [hunk 4]
```diff
@@ -42,6 +44,14 @@ CLASS net/minecraft/class_2586 net/minecraft/block/entity/BlockEntity
 		ARG 2 state
 	METHOD method_31664 setCachedState (Lnet/minecraft/class_2680;)V
 		ARG 1 state
+	METHOD method_38235 toUpdatePacket ()Lnet/minecraft/class_2596;
+		COMMENT Implement and return a packet that should be sent to players nearby when the observable state of
+		COMMENT this block entity changes.
```
Comment (shartte) [inline mappings/net/minecraft/block/entity/BlockEntity.mapping]:
```suggestion
		COMMENT this block entity changes. Return null to not send an update packet.
```

**Review by liach — APPROVED** [review]
:heart:

**Pull request merged**